backends/cadence/cadence.cmake

```diff
@@ -43,6 +43,9 @@ set(CMAKE_CXX_COMPILER ${TOOLCHAIN_HOME}/bin/${CROSS_COMPILE_TARGET}-clang++)
 
 set(CMAKE_C_FLAGS_INIT "-stdlib=libc++ -mtext-section-literals -mlongcalls")
 set(CMAKE_CXX_FLAGS_INIT "-stdlib=libc++ -mtext-section-literals -mlongcalls")
+#workaround for larger compilation time
+SET(CMAKE_CXX_FLAGS_INIT "${CMAKE_CXX_FLAGS_INIT} -fno-strict-aliasing")
```

Comment:
nice! Does this have any side effects?

Reply:
This may cause slight perf degradations in some corner case scenarios. But this is a temporary solution until we figure out proper fix for longer compilation time on some of the files.